package.json

```diff
@@ -32,27 +31,18 @@
     "@babel/preset-env": "^7.4.5",
     "@babel/preset-react": "^7.14.5",
     "@playwright/test": "^1.33.0",
-    "@symfony/webpack-encore": "^4.5.0",
+    "@symfony/webpack-encore": "^3.0.0",
     "@woocommerce/dependency-extraction-webpack-plugin": "^1.7.0",
     "@wordpress/data": "^6.1.5",
     "@wordpress/element": "^4.0.4",
-    "date-and-time": "^0.14.0",
-    "del": "^3.0.0",
     "dotenv": "^16.0.0",
-    "gulp": "^4.0.2",
-    "gulp-cli": "^2.0.1",
-    "gulp-help-doc": "^1.1.1",
-    "gulp-rename": "^2.0",
-    "gulp-zip": "^4.2.0",
-    "minimist": "^1.2.3",
-    "pump": "^3.0.0",
     "sass": "^1.52.1",
     "sass-loader": "^7.0.1",
     "wp-pot": "^1.10.2"
   },
   "scripts": {
     "watch": "webpack --watch",
```

Comment:
@mmaymo Since you are doing this, you can change the watch script to 

`node_modules/.bin/encore dev --watch --env basePath=.`